onnxruntime/contrib_ops/cuda/bert/attention_softmax.cu

```diff
@@ -0,0 +1,900 @@
+#include "contrib_ops/cuda/bert/attention_softmax.h"
+#include "contrib_ops/cpu/bert/attention_common.h"
+#include "core/platform/env_var_utils.h"
+
+using namespace onnxruntime::cuda;
+using namespace cub;
+
+namespace onnxruntime {
+namespace contrib {
+namespace cuda {
+
+struct EnvVarSingleton {
```

Comment:
Rename it to more specified name like SoftmaxEnvVarSingleton?

Reply:
Nice.